core/lib/zksync_core/src/state_keeper/io/mempool.rs

```diff
@@ -455,7 +455,7 @@ impl MempoolIO {
     }
 
     async fn load_previous_l1_batch_hash(&self) -> U256 {
-        tracing::info!(
+        tracing::trace!(
```

Comment:
Dumb question: What's the reason for these changes? Do these logs become spammy with this PR? Or are they already spammy?

Reply:
They are already spammy :)